apis/github

```diff
@@ -103,15 +103,15 @@ Repository.cloneTo = function(self, dest, onProgress)
 		return error("Destination is a file!")
 	end
 
-	for item in self:tree('master'):iter() do
+	for item in self:tree(sha):iter() do
```

Comment:
so `self:tree(sha)` is not invoking `Tree.new()`?

On Fri, Aug 19, 2016, 22:10 Eric Wieser <notifications@github.com> wrote:

> In apis/github
> <https://github.com/eric-wieser/computercraft-github/pull/6#discussion_r75571733>
> :
>
> > @@ -103,15 +103,15 @@ Repository.cloneTo = function(self, dest, onProgress)
> >  		return error("Destination is a file!")
> >  	end
> >
> > -	for item in self:tree('master'):iter() do
> > +	for item in self:tree(sha):iter() do
>
> That's not the same variable! On line 93, that's a local variable.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eric-wieser/computercraft-github/pull/6/files/19c46c2a712742a6fbc420de437ff2993076cbef#r75571733>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA41Fk5ZuueAUEtNrVSEmGBCJX10_DSoks5qhnAQgaJpZM4JoI_u>
> .
>